test/BUILD

```diff
@@ -380,6 +380,14 @@ scala_junit_test(
     print_discovered_classes = True
 )
 
+scala_junit_test(
+    name = "JunitCustomRunner",
+    srcs = ["src/main/scala/scala/test/junit/JunitCustomRunnerTest.scala",
```

Comment:
Can we show an example where the runner is in a stand alone scala_library and then put that in deps? It should work but how to write the junit dependency may not be clear.